src/components/Paragraph/index.js

```diff
@@ -3,14 +3,64 @@ import PropTypes from 'prop-types';
 import React from 'react';
 
 import { useTheme } from 'themes';
+import { bodySmall, bodyMedium, bodyLarge } from 'themes/mixins';
 
-export const Paragraph = ({ children, ...otherProps }) => {
+export const Paragraph = ({
+  children,
+  large,
+  small,
+  inverse,
```

Comment:
As mentioned, I think I'd prefer:
```suggestion
  inverted,
```
for this prop name.